### DISCUSSION TITLE
Hue Outdoor Motion Sensor - Motion Sensitivity

### DISCUSSION THREAD

**Discussion by weimdall** [Koenkk/zigbee2mqtt#13418] [Devices]
Model: 9290030674

Hello,
I have a problem with the Phillips Hue Outdoor Motion Sensor, it is extremely sensitive which make it unreliable.
I tried to lower the "motion_sensitivity" parameter from high to medium, then low .. but it didn't seem to change anything to its actual sensitivity.

Does anyone else encountered this behavior, or is it me ?
Is there a way to test the sensitivity to ensure there no bug anywhere ?

**Comment by EricV-hue** [comment]
Hi, 
I have the same problem. More or less random motion detection. Up to now i thought it was maybe caused by birds
